presto-spi/src/main/java/com/facebook/presto/spi/security/AccessControl.java

```diff
@@ -322,4 +325,6 @@ default AuthorizedIdentity selectAuthorizedIdentity(Identity identity, AccessCon
      * @throws com.facebook.presto.spi.security.AccessDeniedException if not allowed
      */
     void checkCanAddConstraints(TransactionId transactionId, Identity identity, AccessControlContext context, QualifiedObjectName constraintName);
+
+    boolean isSkipPermissionsCheckEnabled();
```

Comment:
have this default to false so accessControls it's not relevant to don't need to implement it.